lib/MetagenomeUtils/Utils/MetagenomeFileUtils.py

```diff
@@ -11,8 +11,10 @@
 
 from DataFileUtil.DataFileUtilClient import DataFileUtil
 from AssemblyUtil.AssemblyUtilClient import AssemblyUtil
-from KBaseReport.KBaseReportClient import KBaseReport
-
+from KBaseReport.KBaseReportClient  import KBaseReport
+from SetAPI.SetAPIClient            import SetAPI
+from biokbase.workspace.client      import Workspace as workspaceService
+from pprint import pprint, pformat
```

Comment:
I don't see pprint is used anywhere. You could remove pprint just leave pformat.